assets/src/edit-story/theme.js

```diff
@@ -47,6 +47,8 @@ const theme = {
     mg: {
       v1: '#616877',
       v2: '#DADADA',
+      v3: '#3C4043',
```

Comment:
Note that this color is not actually used as is. The text in question has an opacity of 0.86 resulting in a color almost identical to `bg.v5`. So I think you can safely replace all instances of `mg.v3` with that and remove it from here.

Reply:
Let's use the real color here (not `bg.v5`, it's subtle but it really does make a difference) however let's try to play with opacity and deduce the color from the existing colors we have and `polished`'s `rgba`.

Reply:
Done